include/ck/tensor_operation/gpu/element/unary_element_wise_operation.hpp

```diff
@@ -355,7 +355,8 @@ struct UnaryDivide
     __host__ __device__ void operator()(T& y, const T& x) const
     {
         static_assert(is_same<T, float>::value || is_same<T, double>::value ||
-                          is_same<T, int32_t>::value,
+                          is_same<T, int32_t>::value || is_same<T, f8_t>::value ||
```

Comment:
We probably should add seperate implementation for fp8. I think we dont support native divide by fp8. (check ConvInvscale)

Reply:
Just do it through fp32 conversion.